client/src/Components/StudentGroup.js

```diff
@@ -7,14 +7,14 @@ class StudentGroup extends Component{
     {
         super(props)
         this.state={
-            leavegroupbtn: true
+            btn_RequestForLeaveGroup: true
```

Comment:
change button name to a new one

Reply:
how about `ableToLeaveGroup`? because you only care if the user can leave the group

Reply:
OK, I will change it then